src/Block_Reservoir.f90

```diff
@@ -1,28 +1,32 @@
-module Block_Reservoir
+Module Block_Reservoir
+
+
 
         implicit none
-        
+
         ! --------------------------------------------------------------------
         !       reservoir to RBM variables
         !----------------------------------------------------------------------
 
-        !------------------------read in reservoir info--------------------------
-        real, dimension (:), allocatable:: dam_lat, dam_lon, res_grid_lat, res_grid_lon
-        real, dimension (:), allocatable:: res_depth_feet, res_width_feet, res_length_feet
+        !------------------------read in reservoir
+        !info--------------------------
+        real, dimension (:), allocatable:: dam_lat, dam_lon, res_grid_lat,res_grid_lon
+        real, dimension (:), allocatable:: res_depth_feet, res_width_feet,res_length_feet
         integer, dimension (:), allocatable:: dam_number, start_operating_year
-        real, dimension (:), allocatable:: res_top_vol, res_bot_vol, res_max_flow, res_min_flow
+        real, dimension (:), allocatable:: res_top_vol, res_bot_vol,res_max_flow, res_min_flow
         integer, dimension (:), allocatable:: res_start_node, res_end_node
-       !  integer, dimension (:,:), allocatable:: nodes_x   !for each reach, what are all the nodes
+       !  integer, dimension (:,:), allocatable:: nodes_x   !for each reach,
+       !  what are all the nodes
         real , dimension(:), allocatable  :: rmile_node
         integer, dimension(:,:), allocatable  :: res_num, nodes_x, x_dist_res
         logical, dimension(:,:), allocatable  :: res_pres, res_upstream
-        logical, dimension(:), allocatable :: flag_turnover 
+        logical, dimension(:), allocatable :: flag_turnover
         logical :: reservoir, res_upstreamx ! the first is TRUE or FALSE in fifth line of _Network file whether reserovirs are present 
         integer, dimension (:), allocatable :: xres, resx
         integer :: xres2, nres, nm_start, ncell0res
-        real, dimension(:), allocatable :: dx_res, dt_res, reservoir_storage,reservoir_storage_prev
+        real, dimension(:), allocatable :: dx_res, dt_res,reservoir_storage,reservoir_storage_prev
         real ::  advec_tot,q_surf_tot
-        real, dimension (:), allocatable::diffusion_tot,advec_hyp_tot,advec_epi_tot, qsurf_tot
+        real, dimension (:),allocatable::diffusion_tot,advec_hyp_tot,advec_epi_tot, qsurf_tot
```

Comment:
In line 27, 29, 45, 47, 50, 52, 55, 56 and 65, add a blank between variable and comma.